Tests/Security/SessionDownloadStrategyTest.php

```diff
@@ -49,40 +47,4 @@ public function testIsGrantedTrue()
         $strategy = new SessionDownloadStrategy($translator, $session, 1);
         $this->assertTrue($strategy->isGranted($media, $request));
     }
-
-    /**
-     * @group legacy
-     * NEXT_MAJOR: remove this method
-     */
-    public function testIsGrantedTrueWithContainer()
-    {
-        $container = $this->getMock('Symfony\Component\DependencyInjection\ContainerInterface');
-
-        $translator = $this->getMock('Symfony\Component\Translation\TranslatorInterface');
-        $media = $this->getMock('Sonata\MediaBundle\Model\MediaInterface');
-        $request = $this->getMock('Symfony\Component\HttpFoundation\Request');
-        $session = $this->getMock('Symfony\Component\HttpFoundation\Session\Session');
-
-        $session->expects($this->any())
-            ->method('get')
-            ->willReturn(0);
-
-        $container->expects($this->once())
-            ->method('get')
-            ->willReturn($session);
-
-        $strategy = new SessionDownloadStrategy($translator, $container, 1);
-
-        $this->assertTrue($strategy->isGranted($media, $request));
-    }
-
-    /**
-     * @expectedException \InvalidArgumentException
-     */
-    public function testInvalidArgumentException()
```

Comment:
this was a test to check if the parameter is either `session` or `container` instance. In the previous commit `NEXT_MAJOR : delete this method` was missing.